src/Features/Core/Portable/ChangeSignature/DelegateInvokeMethodReferenceFinder.cs

```diff
@@ -109,7 +109,7 @@ protected override void FindReferencesInDocument<TData>(
             var convertedType = (ISymbol?)state.SemanticModel.GetTypeInfo(node, cancellationToken).ConvertedType;
             if (convertedType != null)
             {
-                convertedType = SymbolFinder.FindSourceDefinition(convertedType, state.Solution, cancellationToken) ?? convertedType;
```

Comment:
@CyrusNajmabadi Not sure if you recognize this code at all -- the finder API assumes that once we have a single document scanning it is non-async. I didn't spread async beyond here (since this would impact all finders), and also it wasn't clear to me why this shouldn't just be doing symbol equality anyways...

(this class is also hellishly expensive since it's literally looking across all projects and all documents and one-by-one analyzing them...)

Reply:
oh yeah.  this is for ChangeSig and is *awful*.  this is fine.